pkg/urbit/vere/io/term.c

```diff
@@ -664,10 +664,15 @@ _term_io_suck_char(u3_utty* uty_u, c3_y cay_y)
     else {
       if ( (cay_y >= 'a') && (cay_y <= 'z') ) {
         tat_u->esc.ape = c3n;
+        //  XX for backwards compatibility, check kelvin version
+        //  and fallback to [%met @c]
+        //
```

Comment:
@joemfb I don't think these are necessary? Current dill, as it exists on the network, is already forward-compatible with the inputs from the new runtime. See also #4942.

Reply:
But previous dill isn't, so people with old piers could end up in a situation where they have to juggle binaries to get upgraded (or even just poke around). (Since non-modified `%txt` input will still work, they can probably get unstuck by just typing `|merge ...` -- these concerns are a little more theoretical.)

In general, dropping support for old protocols should happen after some longer period time, and it should be straightforward to continue supporting multiple versions. But those patterns are yet to be established, which is why I'm settling for comments.

Reply:
Fair game. I'm relatively comfortable (and relieved (^: ) to merge this as is, but if any poor sap happens to get hurt by this: blame me.